src/objects/zcl_abapgit_object_devc.clas.abap

```diff
@@ -558,4 +621,42 @@ CLASS zcl_abapgit_object_devc IMPLEMENTATION.
 
   ENDMETHOD.
 
+
+  METHOD is_empty.
+
+    DATA: lv_object_name TYPE tadir-obj_name.
+
+    SELECT SINGLE obj_name
+           FROM tadir
+           INTO lv_object_name
+           WHERE obj_name <> iv_package_name
+           AND   devclass = iv_package_name.
```

Comment:
I guess this will break if objects in the package are named the same as the package?

Reply:
you're right. Fixed it.